inno_setup/gnucash-mingw64.iss

```diff
@@ -503,9 +503,15 @@ begin
     // internal exception, you'll fall down to the except block where
     // you expand the {%allusersprofile}
     Folder := ExpandConstant('{userdocs}');
-    // the {userdocs} folder expanding succeded, so let's test if the
-    // folder exists and if not, expand {%allusersprofile}
-    if not DirExists(Folder) then
+    // the {userdocs} folder expanding succeeded, so let's test if the
+    // folder exists and if not, try to expand %userprofile%\Documents.
+    // If this fails, default to the expansion of {%allusersprofile}.
+    if not DirExists(Folder) then begin
+      Folder := ExpandConstant('{%userprofile}') + '\Documents';
+      if not DirExists(Folder) then
+        Folder := '';
+    end;
+    if Folder = '' then
```

Comment:
This is silly. The only way for `Folder = ''` to be true is for it to be set to it at line 512, so just call `ExpandConstant('{%allusersprofile}') there.

Reply:
Style only - but after reversion we use the original version of the proc again.